### ISSUE TITLE
If we clear the database you get a redirect error

### ISSUE THREAD

**Issue by adriaandotcom** [BustByte/coronastatus.no#296]
Not sure if we should fix this, but can image that a record in the future will not be found and it will result in a non working website.

![telegram-cloud-photo-size-4-6041722683193930583-y](https://user-images.githubusercontent.com/1079135/77462142-f9047e80-6e03-11ea-8552-dfd53ec6e187.jpg)

**Comment by fossecode** [comment]
I think this also occurs when the DBs of new instances are wiped right before launch.

**Comment by adriaandotcom** [comment]
Yeah, maybe we should add a redirect count or something and if that hits more than 2 or something we should clear the cookie.

**Comment by michaelmcmillan** [comment]
Just hit this problem as well. Do you know where the loop happens?

**Comment by michaelmcmillan** [comment]
Could also be solved by not redirecting and instead showing a "Invalid pass code" message.

**Comment by adriaandotcom** [comment]
Yes, better to show invalid passcode
